gr-digital/lib/constellation.cc

```diff
@@ -632,6 +707,8 @@ constellation_bpsk::constellation_bpsk()
     d_constellation[1] = gr_complex(1, 0);
     d_rotational_symmetry = 2;
     d_dimensionality = 1;
+    // normalization_t norm = POWER_NORMALIZATION;
+    // normalize(norm);
```

Comment:
Does removing this code cause any behavior changes? I've sometimes found that the default normalization of constellations is not what I want and done some changes by hand (such as multiply my signal by a constant). I wonder if I'd need to tweak my changes due to this.

Reply:
Yes this is very frustrating. I think the default constellation amplitudes make no god damn sense at all which is why you see these lines. The unfortunate thing is there's several QA tests involving equalizers that depend on these bizarre and unscaled constellations. I've just decided that I don't want to open Pandora's box on fixing that and anything else this might break. Even if I think it's the correct solution. Hence I've left it as a comment for now.

Reply:
So to be clear. These had no normalization by default. I thought this was strange and added it. This broke things that I didn't want to fix, so I've left comments as a reminder for the future.